packages/contracts-periphery/tasks/install-drippie-config-multisig.ts

```diff
@@ -0,0 +1,135 @@
+import fs from 'fs'
+
+import { task } from 'hardhat/config'
+import { getChainId } from '@eth-optimism/core-utils'
+
+import { DripConfig, getDrippieConfig, addChecksum } from '../src'
+
+task('install-drippie-config-multisig')
+  .addParam('outfile', 'where to write the bundle JSON file')
```

Comment:
There is no default value, this should not be allowed to be `undefined` because when it tries to write to disk it'll have a bad time

Reply:
Doesn't `addParam` require you to enter a value?